test/js/calc-length.js

```diff
@@ -83,4 +83,74 @@ suite('CalcLength', function() {
     var dividedCalcLength = calcLength.divide(2.5);
     assert.strictEqual(dividedCalcLength.cssString, 'calc(10px+1.28em)');
   });
+
+  test('Adding two CalcLengths returns a new CalcLength with expected values in each value type', function() {
+    var cLength_1 = new CalcLength({px: 15, em: 6})
```

Comment:
shouldn't contract names, so variables should be

var calcLength1
var calcLength2